biome.jsonc

```diff
@@ -7,16 +7,22 @@
     "enabled": true,
     "rules": {
       "recommended": true
-    }
+    },
+    "ignore": ["node_modules", "dist", ".next"]
   },
   "formatter": {
     "enabled": true,
     "indentWidth": 2,
-    "indentStyle": "space"
+    "indentStyle": "space",
+    "ignore": ["node_modules", "dist", ".next"]
   },
   "json": {
+    "parser": {
+      "allowTrailingCommas": true
```

Comment:
Thanks for pointing out the good stuff :D

Reply:
This seems like an unnecessary setting, so I deleted it!
[fix: delete useless config](https://github.com/coauthors/coauthors/pull/5/commits/e85ed745d6e47dc3d71874f87674db6a96cca82f)

Reply:
No. This setting allows you to parse trailing commas in json. If you don't have this setting, the last comma in json will not be formatted when formatting. But if you set this option, the trailing comma in json will be formatted. Try to test it :D

Reply:
Crazy foolishness restore

[chore: restore allowTrailingCommas options](https://github.com/coauthors/coauthors/pull/5/commits/7b842b7259d2a28b309f8716e7a16c836f6412d3)